Sources/Debugging/Debuggable.swift

```diff
@@ -105,11 +105,17 @@ extension Debuggable {
 
 extension String {
     func readableTypeName() -> String {
+    #if swift(>=4.0)
+        let characterSequence = self
+            .split(separator: ".")
```

Comment:
I would end the `#else` after the first line - no point duplicating more code than we have to